docs/software/programming.md

```diff
@@ -74,6 +75,25 @@ YouTube video tutorial: *DietPi Docker Setup on Raspberry Pi 3 B Plus*.
 
 <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/y_VfLOGm5nA" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+## Docker-Compose
+
+Docker Compose is a tool for running multi-container applications on Docker defined using the Compose file format.
+
+![DietPi programming software Docker-Compose](https://raw.githubusercontent.com/docker/compose/master/logo.png)
+
+=== "Update"
+
+    Update to latest version:
+
+    ```
+    sudo pip3 install docker-compose --upgrade
+    ```
+
+***
+
+Official documentation: <https://docs.docker.com/compose>
```

Comment:
Add 2 blanks at the end of this line to get a newline for the following line "Getting started: ..."

Otherwise it looks like:
![grafik](https://user-images.githubusercontent.com/57687516/107012454-534acb80-6799-11eb-80c1-7e6ffffc4853.png)

Reply:
done